pkg/apps/srv/argocd/ingress.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 spec:
   ingressClassName: "nginx"
   rules:
-  - host: argocd.idpbuilder.cnoe.io.local
+  - host: argocd.idpbuilder.cnoe.io.localtest.me
```

Comment:
Does it make sense to use a so long DNS name `*.idpbuilder.cnoe.io.localtest.me` vs `*.idpbuilder.localtest.me` or even better `*.idp.localtest.me` ?